rest_sensors.cpp

```diff
@@ -799,6 +799,39 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                 else if (rid.suffix == RConfigTempThreshold || rid.suffix == RConfigHumiThreshold)
                 {
                 }
+                else if (rid.suffix == RConfigLock)
+                {
+                    bool ok;
+                    
+                    item = sensor->item(RConfigLock);
+                    if (item && map[pi.key()].type() == QVariant::Bool)
+                    {
+                        bool val = map[pi.key()].toBool();
```

Comment:
Just a small suggestion since I'd like to get more const correctness in the code base. `const` helps a lot when reading code, I don't have to think and reason if something is done with a value. `const` also helps to catch errors at compile time if something is changed which isn't meant to be changed.

Here `val` is never changed, so it can be `const val`.

Reply:
done, and have changed var name too